crux_time/src/protocol/instant.rs

```diff
@@ -12,27 +10,27 @@ const NANOS_PER_SEC: u32 = 1_000_000_000;
 /// - seconds: number of seconds since the Unix epoch (1970-01-01T00:00:00Z)
 /// - nanos: number of nanoseconds since the last second
 ///
-/// Note: We cannot use `std::time::Instant` because it is not deserializable.
+/// Note: We cannot use `std::time::Instant` because it is opaque and not deserializable.
```

Comment:
And also semantically not the right thing, if we want to get pedantic 😅

Reply:
Yeah. Good point. I removed the comment altogether